server/game/cards/06-CotE/InDefenseOfRokugan.js

```diff
@@ -0,0 +1,23 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes } = require('../../Constants');
+
+class InDefenseOfRokugan extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Set an attacking character to 0 military skill',
+            condition: () => this.game.isDuringConflict(),
+            cost: ability.costs.sacrifice(card => card.type === CardTypes.Character && card.isDefending()),
+            target: {
+                cardType: CardTypes.Character,
+                cardCondition: card => card.isAttacking(),
+                gameAction: ability.actions.cardLastingEffect({
+                    effect: ability.effects.setMilitarySkill(0)
+                })
+            }
```

Comment:
You still need an 'effect' here - the default for cardLastingEffect is 'X plays Y to apply a lasting effect to Z' which isn't particularly helpful

Reply:
Right I get it now.  This is a bit of a magical string that you just have to know what it does.  I've just pushed a change.